reporter/html.js

```diff
@@ -844,16 +844,27 @@ export function escapeText( s ) {
 	} );
 
 	QUnit.testDone( function( details ) {
-		var testTitle, time, testItem, assertList,
+		var testTitle, time, testItem, assertList, status,
 			good, bad, testCounts, skipped, sourceName,
 			tests = id( "qunit-tests" );
 
 		if ( !tests ) {
 			return;
 		}
 
+
 		testItem = id( "qunit-test-output-" + details.testId );
 
+		if ( details.failed > 0 ) {
+			status = "failed";
+		} else {
+			status = details.skipped ? "skipped" : "passed";
```

Comment:
a test could also be `todo`, defined by `details.todo`.

Reply:
added conditional for todo